pytorch_lightning/utilities/argparse.py

```diff
@@ -184,6 +221,8 @@ def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
         else:
             use_type = arg_types[0]
 
+        # TODO: Defer these to an optional static method, like
+        # `argparse_hacks()`?
```

Comment:
I'd rather not put this TODO as it will be easily forgotten. Feel free to open a PR for it in the future!

Reply:
Done in b385eefb